tests/test_on_data/test_gin.py

```diff
@@ -52,39 +56,78 @@
 def custom_name_func(testcase_func, param_num, param):
     return (
         f"{testcase_func.__name__}_{param_num}_"
-        f"{parameterized.to_safe_name(param.kwargs['recording_interface'].__name__)}"
+        f"{parameterized.to_safe_name(param.kwargs['data_interface'].__name__)}"
     )
 
 
 class TestNwbConversions(unittest.TestCase):
     savedir = Path(tempfile.mkdtemp())
 
+    parameterized_lfp_list = [
+        param(
+            data_interface=AxonaLFPDataInterface,
+            interface_kwargs=dict(file_path=str(DATA_PATH / "axona" / "dataset_unit_spikes" / "20140815-180secs.eeg")),
+        ),
+        param(
+            data_interface=SpikeGLXLFPInterface,
+            interface_kwargs=dict(
+                file_path=str(
+                    DATA_PATH / "spikeglx" / "Noise4Sam_g0" / "Noise4Sam_g0_imec0" / "Noise4Sam_g0_t0.imec0.lf.bin"
+                )
+            ),
+        ),
+    ]
+
+    @parameterized.expand(input=parameterized_lfp_list, name_func=custom_name_func)
+    def test_convert_lfp_to_nwb(self, data_interface, interface_kwargs):
+        nwbfile_path = str(self.savedir / f"{data_interface.__name__}.nwb")
+
+        class TestConverter(NWBConverter):
+            data_interface_classes = dict(TestLFP=data_interface)
+
+        converter = TestConverter(source_data=dict(TestLFP=interface_kwargs))
+        for interface_kwarg in interface_kwargs:
+            if interface_kwarg in ["file_path", "folder_path"]:
+                self.assertIn(member=interface_kwarg, container=converter.data_interface_objects["TestLFP"].source_data)
+        converter.run_conversion(nwbfile_path=nwbfile_path, overwrite=True)
+        recording = converter.data_interface_objects["TestLFP"].recording_extractor
+        with NWBHDF5IO(path=nwbfile_path, mode="r") as io:
+            nwbfile = io.read()
+            nwb_lfp_unscaled = nwbfile.processing["ecephys"]["LFP"]["ElectricalSeries_lfp"].data
+            nwb_lfp_conversion = nwbfile.processing["ecephys"]["LFP"]["ElectricalSeries_lfp"].conversion
+            # Technically, check_recordings_equal only tests a snippet of data. Above tests are for metadata mostly.
+            # For GIN test data, sizes should be OK to load all into RAM even on CI
+            npt.assert_array_equal(x=recording.get_traces(return_scaled=False).T, y=nwb_lfp_unscaled)
+            npt.assert_array_almost_equal(
+                x=recording.get_traces(return_scaled=True).T * 1e-6, y=nwb_lfp_unscaled * nwb_lfp_conversion
+            )
+
     parameterized_recording_list = [
         param(
-            recording_interface=NeuralynxRecordingInterface,
+            data_interface=NeuralynxRecordingInterface,
```

Comment:
why change these?

Reply:
So the `custom_name_function` applies to all 3 interface types. Right now it's only for `recording_interface` keywords.

Reply:
oh, ok that makes sense